src/utils/utils.js

```diff
@@ -199,17 +199,45 @@ module.exports = {
 
     // Example: 0.6.5-c0-d1
     // c0 = contract version 0, d1 = db version 1
-    const versionDir = `${regexGroups[4]}_${regexGroups[5]}`; // c0_d1
+    const versionDir = `${basePath}/${regexGroups[4]}_${regexGroups[5]}`; // c0_d1
+
+    // Create data dir if needed
+    fs.ensureDirSync(versionDir);
+
+    return versionDir;
+  },
+
+  /*
+  * Returns the path where the blockchain data directory is, and also creates the directory if it doesn't exist.
+  */
+  getBlockchainDataDir: () => {
+    const versionDir = getBlockchainVersionDir();
 
     // production
-    const dataDir = `${basePath}/${versionDir}/nedb`;
+    const dataDir = `${versionDir}/nedb`;
 
     // Create data dir if needed
     fs.ensureDirSync(dataDir);
 
     return dataDir;
   },
 
+/*
+  * Returns the path where the blockchain log directory is, and also creates the directory if it doesn't exist.
+  */
+  getLogDir: () => {
+    const versionDir = getBlockchainVersionDir();
+
+    // production
```

Comment:
can remove this comment. technically it can be for dev too since the `dev` folder is added in `getBlockchainVersionDir`